### ISSUE TITLE
7.03 SSR broken = window is not defined

### ISSUE THREAD

**Issue by luisatmaniak** [ankurk91/vue-flatpickr-component#89]
**I'm submitting a ...**  (check one with "x")
```
[ x] Bug report => search github for a similar issue or PR before submitting
[ ] Feature request
[ ] Other, please describe
```

**Tell about your platform**
* flatPickr version : 4.x.x
* Vue.js version : 2.x.x
* Browser name and version : Chrome|Firefox|Safari x.x.x
* This package version : 7.0.3

**Current behavior**
Not SSR compatible since 7.0.3, webpack generates "})(window, function(__WEBPACK_EXTERNAL_MODULE__0__) {" on line 10 of vue-flatpickr-component/dist/vue-flatpickr.js, which throws an error server-side.

**Expected behavior**
No to throw an error during SSR.

**Minimal reproduction of the problem with instructions**
Import it from node, don't even need to use it.

**Comment by ankurk91** [comment]
https://github.com/webpack/webpack/issues/6642

**Comment by ankurk91** [comment]
Thanks for reporting.

Should be fixed in v7.0.4.
Feel free to report again if you found it broken.
